src/gui/base/WizardDialog.ts

```diff
@@ -175,7 +195,7 @@ class WizardDialogAttrs<T> {
 			type: ButtonType.Secondary,
 		}
 		return {
-			left: [backButtonAttrs],
+			left: currentPageIndex >= 0 && this.allowedToVisitPage(currentPageIndex - 1, currentPageIndex) ? [backButtonAttrs] : [],
```

Comment:
is `>= 0`  correct here? I guess for 0 the back button is actually a close button?

Reply:
I decided on using "Cancel" since this was also used before in this place.